requirements/minimal_requirements.txt

```diff
@@ -14,6 +14,7 @@ pytorch_lightning==1.7.1
 torch==1.11.0
 torchvision==0.12.0
 pillow==7.1.2
+packaging
```

Comment:
Let's put a version here and not leave it open as we otherwise don't test "minimal" requirements. Most other dependencies are from 2021. So maybe we can just use version "21.0"?

Reply:
In TorchGeo, when we had a dependency on packaging, we used packaging>=17, as 17 was the minimum version of packaging supported by our minimum version of lightning. Like you said, the version comparison/parsing stuff has always been there, it's more about matching your other minimum requirement versions.